content/guides/10-minute-tutorial.md

```diff
@@ -792,12 +793,281 @@ Feature: Is it Friday yet?
+```gherkin
+Feature: Is it Friday yet?
+  Everybody wants to know when it's Friday
+
+  Scenario Outline: Today is or is not Friday
+    Given today is <day>
+    When I ask whether it's Friday yet
+    Then I should be told <answer>
+
+  Examples:
+    | day | answer |
+    | "Friday" | "TGIF" |
+    | "Sunday" | "Nope" |
+    | "anything else!" | "Nope" |
+```
+
+{{% block "javascript" %}}
+Update the `stepdefs.js` file to use the `<day>` and `<answer>` variables.
+```javascript
+const assert = require('assert');
+const { Given, When, Then } = require('cucumber');
+
+function isItFriday(today) {
+  return (today === "Friday") ? "TGIF" : "Nope";
+}
+
+Given('today is {string}', function (givenDay) {
+  this.today = givenDay;
+});
+
+When('I ask whether it\'s Friday yet', function () {
+  this.actualAnswer = isItFriday(this.today);
+});
+
+Then('I should be told {string}', function (expectedAnswer) {
+  assert.equal(this.actualAnswer, expectedAnswer);
+});
+
+```
+{{% /block %}}
+
+{{% block "ruby" %}}
+Update the `stepdefs.rb` file to use the `<day>` and `<answer>` variables.
+```ruby
+module FridayStepHelper
+  def is_it_friday?(day)
+    day == 'Friday' ? 'TGIF' : 'Nope'
+  end
+end
+World FridayStepHelper
+
+Given("today is {string}") do |given_day|
+  @today = given_day
+end
+
+When("I ask whether it's Friday yet") do
+  @actual_answer = is_it_friday?(@today)
+end
+
+Then("I should be told {string}") do |expected_answer|
+  expect(@actual_answer).to eq(expected_answer)
+end
+```
+{{% /block %}}
+
+Run Cucumber again:
+
+{{% block "javascript" %}}
+```shell
+.........
+
+3 scenarios (3 passed)
+9 steps (9 passed)
+0m00.001s
+```
+{{% /block %}}
+
+{{% block "ruby" %}}
+```ruby
+Feature: Is it Friday yet?
+  Everybody wants to know when it's Friday
+
+  Scenario Outline: Today is or is not Friday # features/is_it_friday_yet.feature:4
+    Given today is <day>                      # features/is_it_friday_yet.feature:5
+    When I ask whether it's Friday yet        # features/is_it_friday_yet.feature:6
+    Then I should be told <answer>            # features/is_it_friday_yet.feature:7
+
+    Examples:
+      | day              | answer |
+      | "Friday"         | "TGIF" |
+      | "Sunday"         | "Nope" |
+      | "anything else!" | "Nope" |
+
+3 scenarios (3 passed)
+9 steps (9 passed)
+0m0.021s
+```
+{{% /block %}}
+
 # Summary
 
-In this brief tutorial you've seen how to install Cucumber, and how to follow
-the BDD process to develop a very simple function. The next natural steps would
-be:
+{{% block "javascript" %}}
+In this brief tutorial you've seen how to install Cucumber, how to follow
+the BDD process to develop a very simple function, and how to use that function to evaluate multiple scenarios!
+{{% /block %}}
 
-* Move the `{{% text "java" %}}isItFriday{{% /text %}}{{% text "javascript" %}}isItFriday{{% /text %}}{{% text "ruby" %}}is_it_friday{{% /text %}}` function out from the test code into production code (refactoring)
```

Comment:
I'd like to keep this somehow, to clarify that this is what we're moving on with. (Did we explicitly move the function from test code to production code, for instance?